setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import find_packages
```

Comment:
If we use python3.11 we can just use pyproject.toml

This should probably be in another project (e.g., json-playground) with all the rest of the code. In the meantime, this repo should reference the hooks repo.